front/app/components/EsriMap/utils.tsx

```diff
@@ -5,6 +5,9 @@ import VectorTileLayer from '@arcgis/core/layers/VectorTileLayer';
 import WebTileLayer from '@arcgis/core/layers/WebTileLayer';
 import Layer from '@arcgis/core/layers/Layer';
 
+// utils
+import debounceFn from 'lodash/debounce';
+
```

Comment:
@luucvanderzee I'm getting a dependency cruiser error with this, but it seems like it's working properly? :thinking: Do you know why we'd have a rule restricting the import of the debounceFn here?